test/unit/serializing_test.rb

```diff
@@ -125,6 +125,14 @@ def test_update_data_only_includes_relationship_data
     expected = {
       "type" => "articles",
       "id" => "1",
+      "relationships"=> {
+        "author"=> {
+          "data"=>{
+            "type"=>"people",
+            "id"=>"9"
+          }
+        }
+      },
```

Comment:
I don't think that in this case relationships should be serialized
`#as_json_api` is used for gathering a payload that will be sent to server
and we should not add to payload relationships that weren't changed by user 
example:
```ruby
post = Post.includes(:author).find(1)
post.title = 'changed title'
post.save # author relationship should not be present in payload
```

Reply:
Also, #318 does not address the issues with creation

Reply:
@fzf can you please provide some code snippet that explains *issue with creation* that you have encountered?

Reply:
Relationship data is never actually sent, it is cleared before it can be sent. Here is a spec showing that it is a false positive. https://github.com/JsonApiClient/json_api_client/pull/319

Reply:
#319 had a naming issue you can see the failure now @senid231